### PR TITLE
Update 2018.md

### PR DESCRIPTION
**Pull request by mediatinker**
added section for workshops. Added 3 announcements.

**Event title:**

**Organizer's name:**

cc @hasgeek/events-review

### CODE REVIEW

File: _50p_events/2018.md [hunk 1]
```diff
@@ -76,6 +84,13 @@ overview:
  right_content: |
   ![Keep yourself updated on fintech](https://images.hasgeek.com/embed/file/38853704ae224fba8cde9194d153190e)
 
+workhops:
```
Comment (akshaybhalotia) [inline _50p_events/2018.md]: Typo

**Comment by akshaybhalotia** [comment]
@mediatinker Check this file, line 32 onwards: https://github.com/hasgeek/events/blob/master/_reactfoo_events/2017.md
We put it as `related_events`.

**Comment by mediatinker** [comment]
I truly do not care what you call it, but these workshops are integrated into the 50p schedule. I think the ReactFoo ones were more separate.

**Comment by akshaybhalotia** [comment]
Agree, but there's nothing built right now to represent it that way and it's just syntactic. The representation is by cards which take you to their independent pages with more description.

**Comment by mediatinker** [comment]
Please do whatever needs to be done to make these things appear on the site. I don't know the conventions of this system well enough yet. Sorry.

**Comment by akshaybhalotia** [comment]
Okay

**Pull request merged**